api/logs/views.py

```diff
@@ -0,0 +1,216 @@
+from modularodm import Q
+from rest_framework import generics, permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+
+from website.models import Node, NodeLog
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from api.base.filters import ODMFilterMixin
+from api.base.utils import get_user_auth
+from api.base import permissions as base_permissions
+from api.logs.serializers import NodeLogSerializer
+from api.nodes.serializers import NodeSerializer
+from api.nodes.utils import get_visible_nodes_for_user
+
+class LogList(generics.ListAPIView, ODMFilterMixin):
```

Comment:
I'm not sure if it makes sense to include this in the API at this point.

1. It's a very expensive endpoint. On production data, this endpoint took 7.5 seconds to return a response.
2. We don't have a use case for this.